<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalDataForm</class>
 <widget class="QWidget" name="PersonalDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Введите имя пользователя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="genderHorizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="genderGroupBox">
       <property name="title">
        <string>Выберите пол</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="maleRadioButton">
          <property name="text">
           <string>Мужской</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="femaleRadioButton">
          <property name="text">
           <string>Женский</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="birtdateHorizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QDateEdit" name="birthdateEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="professionHorizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="professionComboBox">
       <property name="toolTip">
        <string>Если нет подходящей профессии - введите свою</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="assessmentHorizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QSpinBox" name="assessmentSpinBox">
       <property name="toolTip">
        <string/>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="userExcludedFromAsstimatoionLabel">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Пользователь исключён из оценивания</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Имя</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="genderLabel">
     <property name="text">
      <string>Пол</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="birthdateLabel">
     <property name="text">
      <string>Дата рождения</string>
     </property>
     <property name="buddy">
      <cstring>birthdateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="professionLabel">
     <property name="text">
      <string>Профессия</string>
     </property>
     <property name="buddy">
      <cstring>professionComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="assessmentLabel">
     <property name="text">
      <string>Оценка</string>
     </property>
     <property name="buddy">
      <cstring>assessmentSpinBox</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>maleRadioButton</tabstop>
  <tabstop>femaleRadioButton</tabstop>
  <tabstop>birthdateEdit</tabstop>
  <tabstop>professionComboBox</tabstop>
  <tabstop>assessmentSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
